examples/search-agent-for-e-commerce/src/app/actions.tsx

```
@@ -50,170 +53,182 @@ export async function continueConversation(
-            temperature: 0,
-            system: `Generate response to user question ${summary} based on the context ${generalInfo}. Your answer begins with: "While we wait for the human assistant,`,
-            messages: [
-              {
-                role: "user",
-                content: summary,
-              },
+        clothes_fitting: {
+          description:
+            "Send to user link to guidelines for clothes fitting https://images.app.goo.gl/LECaeXJfXa7gzYCC8 ",
+          parameters: z.object({}),
+          generate: async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)